pkg/cmd/server/start.go

```diff
@@ -411,21 +411,21 @@ func start(cfg *config, args []string) error {
 			osClientConfigTemplate := kclient.Config{Host: cfg.MasterAddr.URL.String(), Version: latest.Version}
 
 			// Openshift client
-			if osmaster.OSClientConfig, err = ca.MakeClientConfig("openshift-client", osClientConfigTemplate); err != nil {
```

Comment:
tweak the signature of this to pass in provider, username, and groups as three params... I think we'll eventually want to embed groups in the certs, and (for now) we'll leave the directory providerless